src/config/AppPermissions.js

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+/*
+ * Defines the mapping between:
+ * - roles application defined in core Enterprise Application
+ * - WebApplication persona
+ */
+const APP_ROLES = {
+  organizationAdmin: 'Organization.Admin',
+  organizationCollaborator: 'Organization.Collaborator',
+  organizationModeler: 'Organization.Modeler',
+  // powerUser
+  organizationUser: 'Organization.User',
+  // viewer
+  organizationViewer: 'Organization.Viewer',
+  platformAdmin: 'Platform.Admin',
+};
+
+/*
+ * Defines permissions on WebApplication actions
+ */
+export const PERMISSIONS = {
+  canCreateScenario: 'can-create-scenario',
+  canDeleteScenario: 'can-delete-scenario',
+  canEditOrLaunchScenario: 'can-edit-launch-scenario',
+};
```

Comment:
Same remark as above, if integrators are not likely to change these values, we should probably move them elsewhere

Reply:
Changes have been done regarding our discussion ;)